src/tests/ftest/nvme/pool_exclude.yaml

```diff
@@ -33,36 +33,42 @@ pool:
   nvme_size: 300000000000
   svcn: 4
   control_method: dmg
-  rebuild_timeout: 120
+  rebuild_timeout: 180
   pool_query_timeout: 30
 container:
   type: POSIX
   control_method: daos
   properties: cksum:crc16,cksum_size:16384,srv_cksum:on,rd_fac:2
-  oclass: RP_3G6
+ior_flags_common: &ior_flags_common
+  write_flags: "-w -F -k -G 1"
+  read_flags: "-F -r -R -k -G 1"
+  api: DFS
+  ior_test_sequence:
+    #   - [scm_size, nvme_size, transfersize, blocksize]
+    #    The values are set to be in the multiples of 10.
+    #    Values are appx GB.
+    - ["NA", "NA", 4000, 50000000]          # [4K, 51M]
+    - ["NA", "NA", 1000000, 500000000]      # [1M, 512M]
+    - ["NA", "NA", 1000000000, 8000000000]  # [1G, 8G]
 ior:
   client_processes:
     np: 48
   test_file: /testFile
   repetitions: 10
-  dfs_destroy: false
-  iorflags:
-    write_flags: "-w -F -k -G 1"
-    read_flags: "-F -r -R -k -G 1"
-    api: DFS
-    dfs_oclass: RP_3G6
-    dfs_dir_oclass: RP_3G6
-    ior_test_sequence:
-      #   - [scm_size, nvme_size, transfersize, blocksize]
-      #    The values are set to be in the multiples of 10.
-      #    Values are appx GB.
-      - [50000000000, 300000000000, 4000, 50000000]          # [4K, 51M]
-      - [50000000000, 300000000000, 1000000, 500000000]      # [1M, 512M]
-      - [50000000000, 300000000000, 1000000000, 8000000000]  # [1G, 8G]
-test_obj_class:
-  oclass:
-    - RP_3G6
-    - RP_4G1
+  dfs_destroy: False
+  iorflags: !mux
+    rp_3g6:
+      <<: *ior_flags_common
+      dfs_oclass: RP_3G6
+      dfs_dir_oclass: RP_3G6
```

Comment:
Any specific reason to keep these as `G6` instead of `GX` like the other variants below?

Reply:
RP_3G6 is the existing oclass in CI.

Reply:
Right, but the new variants use `GX`, so I wonder if this should also use `GX`. (I don't know)

Reply:
Added @Ravi and @Phil review if update to GX is needed.

Reply:
Might be a question for a developer. The difference is that `GX` will allow DAOS to calculate the maximum group number based on the data, parity, and number of system targets.

Reply:
I think you can use GX... It shouldn't make a difference.

Reply:
Added @Samir for review.

Reply:
BTW not a blocker for me, since it's pre-existing